packages/salesforcedx-vscode-core/test/vscode-integration/commands/forceAuthDevHub.test.ts

```diff
@@ -5,13 +5,18 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
+import { ConfigFile } from '@salesforce/core';
 import { expect } from 'chai';
+import { ConfigSource, OrgAuthInfo } from '../../../src/util/index';
 import {
   createExecutor,
   ForceAuthDevHubDemoModeExecutor,
   ForceAuthDevHubExecutor
 } from '../../../src/commands/forceAuthDevHub';
+import { DEFAULT_DEV_HUB_USERNAME_KEY, SFDX_PROJECT_FILE } from '../../../src/constants';
 import { nls } from '../../../src/messages';
+import { SinonStub, stub, spy } from 'sinon'; //
+import Sinon = require('sinon');
```

Comment:
Extra slashes at the end, I don't think these imports are correctly ordered? Please remove imports that aren't being used as well 🙂